### PR TITLE
Crop Native image Size

### PR DESCRIPTION
**Pull request by Gaojianli**
There is no need to keep a jar or a jvm in the native image. For those who want to use jar, they can just pick another image.

This PR do the following works:
1. Replace the base image of the native image from `eclipse-temurin` to `debain:slim`
2. Introduce an experimental alpine image, which size is even 50% smaller than the debian one.
image

All the two images are tested on my machine.

## Summary by CodeRabbit

- **New Features**
    - Added support for building and pushing Docker images for Alpine Linux.
    - Updated the Docker setup to use Debian 12-slim as the base image.
    - Introduced a new entry point script for Docker, enhancing the launch process of PeerBanHelper.

### CODE REVIEW

**Comment by Ghost-chu** [comment]
考虑是否可与被revert的 https://github.com/Ghost-chu/PeerBanHelper/pull/48 合并

**Comment by Gaojianli** [comment]
> 考虑是否可与被revert的 #48 合并

可能不太行，我不咋熟github的ci，而且改了ci我也没法测

File: .github/workflows/native.yml [hunk 1]
```diff
@@ -58,6 +58,8 @@ jobs:
             type=ref,event=pr
             type=semver,pattern={{version}}-native-linux
```
Comment (Ghost-chu) [inline .github/workflows/native.yml:59]: 标签行为可能需要对不同类型的image分开打

File: .github/workflows/native.yml [hunk 2]
```diff
@@ -58,6 +58,8 @@ jobs:
...
             type=semver,pattern={{major}}.{{minor}}-native-linux
+            type=semver,pattern={{version}}-native-linux-alpine
+            type=semver,pattern={{major}}.{{minor}}-native-linux-alpine
```
Comment (Ghost-chu) [inline .github/workflows/native.yml:62]: 标签行为可能需要对不同类型的image分开打

**Comment by Gaojianli** [comment]
已回滚相关改动

**Review by Ghost-chu — APPROVED** [review]

**Comment by Ghost-chu** [comment]
LGTM

**Pull request merged**